core/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -372,35 +376,14 @@ abstract class AppBase(
     }
   }
 
-  protected def probNotDataset: mutable.HashMap[Long, Set[String]] = {
+  protected def probNotDataset: mutable.HashMap[Long, mutable.LinkedHashSet[String]] = {
```

Comment:
this already has an import:

` import scala.collection.mutable.{ArrayBuffer, HashMap, HashSet}
`

Unless we have immutable somehwere it would be more consistent to just add it to import and remove mutable. from it.. Or change all of them to have mutable.

Reply:
Fixed a couple of files modified in this PR. I did not fix all occurrences because this would bloat the PR.
Later, we can define which coding style and make sure that all the files are consistent. I did not